main.js

```diff
@@ -1,57 +1,75 @@
-const form = document.querySelector('#form');
-const mainHolder = document.querySelector('.container');
-const bookInput = document.querySelector('#book');
+/* eslint-disable  max-classes-per-file */
+const bookList = document.querySelector('#book-list');
+const bookInput = document.querySelector('#title');
 const authorInput = document.querySelector('#author');
 const buttonAdd = document.querySelector('#add');
 
-let books = [];
-
-function show() {
-  const books = JSON.parse(localStorage.getItem('books'));
-  for (let index = 0; index < books.length; index += 1) {
-    bookInput.value = books[index].book;
-    authorInput.value = books[index].author;
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
 }
-form.addEventListener('load', () => {
-  show();
-});
 
-if (localStorage.getItem('books')) {
-  books = JSON.parse(localStorage.getItem('books'));
-}
+class SavelocalItems {
+  static getBook() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
 
-function addBook(book, author) {
-  const newBook = { book, author };
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
+  static saveItem(book) {
+    const books = SavelocalItems.getBook();
+    books.push(book);
```

Comment:
- [ ] [OPTIONAL] I think it would be great to use array methods that do not mutate the original array such as `concat` or the spread operator. 🥸️

**Example**
```
books = [...books, book];
```

**Or**
```
books = books.concat(book);
```